web_console/components/Header.jsx

```diff
@@ -97,14 +102,15 @@ export default function Header() {
   const title = isAdmin ? 'Admin' : 'Fedlearner';
   const navs = isAdmin
     ? [
-      { label: 'Federations', value: 'federation' },
-      { label: 'Users', value: 'user' },
+      { label: 'Federations', value: '/admin/federation' },
+      { label: 'Users', value: '/admin/user' },
     ]
     : [
-      { label: 'Overview', value: 'overview' },
-      { label: 'Jobs', value: 'job' },
-      { label: 'Tickets', value: 'ticket' },
+      { label: 'Overview', value: '/' },
+      { label: 'Jobs', value: '/job/list' },
```

Comment:
what about using a shorter `/job` and `index.jsx` for the frontpage as convention?